styles/controllers/grid/queries.less

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file styles/controllers/gridd/queries.less
```

Comment:
(I'll deal with this rather than having bothering you for a 1-byte change, @NateWr)